apps/pie-microsite/src/_11ty/shortcodes/tokensTable/tokenTypes/colour.js

```diff
@@ -65,13 +65,23 @@ const buildColorExample = token => {
     return `<div class="${classes.join(' ')}" style="${cssVariable}";></div>`;
 };
 
-const buildColorDescription = (token, tokenMetadata) => {
+/**
+ * Builds the overall token description element for global and alias Colour tokens.
+ * @param {*} tokenMetadata the metadata for the token. data such as descriptions
+ * @returns {string} - the description HTML string
+ */
+const buildColorDescription = tokenMetadata => {
     let description = '';
-    if (tokenMetadata.description) {
+
+    if (tokenMetadata.globalToken) {
+        // It's an alias colour token
```

Comment:
i think you meant the other way around?

Reply:
it's the correct way around but I've added a comment to give better context :)